### PR TITLE
Update degraded-environment variable listing

### PR DESCRIPTION
**Pull request by Veykril**
image

### CODE REVIEW

File: ferrocene/doc/safety-manual/src/degraded-environment.rst [hunk 1]
```diff
@@ -24,35 +24,30 @@ undefined code generation. Therefore, the correctness of the Ferrocene
...
+The environment variables listed below were identified as potential sources of
+errors as they may lead to incorrect behavior of the Ferrocene toolchain. Users
+must verify that the values of these variables have not been set prior to using
+the Ferrocene toolchain.
+
+The following environment variables may not be set prior to using the Ferrocene
+toolchian.
 
-.. list-table:: Output Environment Variables
+.. list-table:: Forbidden Environment Variables
```
Comment (pietroalbini) [inline ferrocene/doc/safety-manual/src/degraded-environment.rst:49]: Should we replace this with a single list?

File: ferrocene/doc/safety-manual/src/degraded-environment.rst [hunk 2]
```diff
@@ -24,35 +24,30 @@ undefined code generation. Therefore, the correctness of the Ferrocene
...
-errors as they may lead to incorrect behavior of the Ferrocene toolchain. Users
-must verify that the values of these variables have not been set prior to using
-the Ferrocene toolchain.
-
-The following environment variables control or specify the inputs to the
+The following environment variables control or specify behavior of the
 compiler.
 
-.. list-table:: Input Environment Variables
+.. list-table:: Configuration Environment Variables
```
Comment (pietroalbini) [inline ferrocene/doc/safety-manual/src/degraded-environment.rst:30]: We recently removed table names.

File: ferrocene/doc/safety-manual/src/degraded-environment.rst (hunk 1 above)
Comment (Veykril) [reply ferrocene/doc/safety-manual/src/degraded-environment.rst:49]: You mean combining the two tables and then forbid the latter two vars in their description?

Comment (pietroalbini) [reply ferrocene/doc/safety-manual/src/degraded-environment.rst:49]: Oh, I mean, replacing the second table with a bullet list (not sure why my brain wrote "single" there).

**Review by pietroalbini — APPROVED** [review]
bors merge

**Pull request merged**